src/sentry/tasks/post_process.py

```diff
@@ -66,7 +74,8 @@ def _should_send_error_created_hooks(project):
     return result
 
 
-def _capture_stats(event, is_new):
+def _capture_stats(job: PostProcessJob) -> None:
+    event, is_new = job["event"], job["group_state"]["is_new"]
```

Comment:
Might wanna be careful here when you get `is_new` bool from `job["group_state"]["is_new"]`. I believe for error events we pass in a `group_state` same as the signature in `post_process_group(is_new, is_regression, is_new_group_environment, ..)`, but I'm not 100% sure on that.

Reply:
https://github.com/getsentry/sentry/blob/master/src/sentry/event_manager.py#L1097
 looks the same to me but I guess a new type could be different? for the `group_state` for error events I'm creating that myself from the method params